.changeset/tall-taxis-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Add `element:techdocs/entity-content-empty-state` extension to allow overriding the empty state for the entity page techdocs tab.
```

Comment:
...or maybe `empty-state:techdocs/entity-content`?

Reply:
Yep it feels pretty likely to end up at `empty-state:techdocs/entity-content` just because it's informative. If you want to you could switch it to that either way since they seem about as likely to end up as the final API 🤷

Reply:
Done in c4e4c0f6b08a.